backend/src/common/util/execution_client.go

```diff
@@ -87,7 +87,7 @@ func NewExecutionClientOrFatal(execType ExecutionType, initConnectionTimeout tim
 	case ArgoWorkflow:
 		var argoProjClient *argoclient.Clientset
 		operation := func() error {
-			restConfig, err := rest.InClusterConfig()
+			restConfig, err := GetKubernetesConfig()
```

Comment:
let's do it for the tekton client below as well [[1]](https://github.com/kubeflow/pipelines/blob/7ee7f755b96e39da94dbda8c4bc94d0d0abe527e/backend/src/common/util/execution_client.go#L110)